DurableFunctionsAnalyzer/DurableFunctionsAnalyzer/DurableFunctionsAnalyzer/Resources.resx

```diff
@@ -0,0 +1,281 @@
+  </data>
+  <data name="EntityInterfaceContentAnalyzerDescription" xml:space="preserve">
+    <value>An Entity Interface must only contain methods and must contain at least one method.</value>
+  </data>
+  <data name="EntityInterfaceContentAnalyzerMessageFormat" xml:space="preserve">
+    <value>Interface contains content other than methods
+ ('{0}')</value>
+  </data>
+  <data name="EntityInterfaceContentAnalyzerNoMethodsMessageFormat" xml:space="preserve">
+    <value>Interface doesn't contain any methods and must contain at least one.</value>
+  </data>
+  <data name="EntityInterfaceContentAnalyzerTitle" xml:space="preserve">
+    <value>An Entity Interface must only contain methods and must contain at least one method.</value>
+  </data>
+  <data name="EntityInterfaceParameterAnalyzerDescription" xml:space="preserve">
+    <value>An Entity Interface must only have methods with at most one parameter</value>
+  </data>
+  <data name="EntityInterfaceParameterAnalyzerMessageFormat" xml:space="preserve">
+    <value>Method '{0}' has more than one parameter</value>
+  </data>
+  <data name="EntityInterfaceParameterAnalyzerTitle" xml:space="preserve">
+    <value>An Entity Interface must only have methods with at most one parameter</value>
+  </data>
+  <data name="EntityInterfaceReturnTypeAnalyzerDescription" xml:space="preserve">
+    <value>An Entity Interface must only have methods that return void, Task, or Task&lt;T&gt;</value>
+  </data>
+  <data name="EntityInterfaceReturnTypeAnalyzerMessageFormat" xml:space="preserve">
+    <value>Method '{0}' has a return type '{1}'; return type should only be void, Task, or Task&lt;T&gt;</value>
+  </data>
+  <data name="EntityInterfaceReturnTypeAnalyzerTitle" xml:space="preserve">
+    <value>An Entity Interface must only have methods that return void, Task, or Task&lt;T&gt;</value>
+  </data>
+  <data name="EntityStaticAnalyzerDescription" xml:space="preserve">
+    <value>Entity functions must be static</value>
+  </data>
+  <data name="EntityStaticAnalyzerMessageFormat" xml:space="preserve">
+    <value>Function '{0}' is not marked static</value>
+  </data>
+  <data name="EntityStaticAnalyzerTitle" xml:space="preserve">
+    <value>Entity functions must be static</value>
+  </data>
+  <data name="EnvironmentVariableAnalyzerTitle" xml:space="preserve">
+    <value>Environment Variables must be accessed in a deterministic way inside an orchestrator</value>
+  </data>
+  <data name="GuidAnalyzerTitle" xml:space="preserve">
+    <value>Guid calls must be deterministic inside an orchestrator</value>
+  </data>
+  <data name="IOTypesAnalyzerTitle" xml:space="preserve">
+    <value>I/O operations are not allowed inside an orchestrator</value>
+  </data>
+  <data name="MethodAttributeAnalyzerTitle" xml:space="preserve">
+    <value>Method calls in an orchestrator must be marked [Deterministic] above the method declaration</value>
```

Comment:
I would remove "above the method declaration". The C# compiler allows it to be above or even to the left of the method declaration.

Reply:
Ok, I'll change it to "on the method declaration." This message will be displayed on the method invocation expression inside the orchestrator function.